hooks/image/54-ek-launchers

```diff
@@ -0,0 +1,12 @@
+# Expose Endless Key launchers in the app grid (once a specific content pack has been downloaded)
+
+if [[ ! "${EIB_FLATPAK_REMOTE_FLATHUB_APPS}" =~ .*"org.endlessos.Key".* ]]; then
+  exit 0
+fi
+
+cat <<- EOF > /etc/systemd/user-environment-generators/62-endless-key.sh
+  #!/bin/bash
```

Comment:
Does `<<-` strip all leading whitespace, or only leading tabs?

Reply:
Ah, the docs say tabs. Good catch. Yay invisible deterministic whitespace!